src/components/NewProjectFlow/flow/SelectBatches.tsx

```diff
@@ -104,7 +104,15 @@ export default function SelectBatches(props: SelectBatchesProps): JSX.Element |
         });
       }
 
-      return NurseryBatchService.getAllBatches(organizationId, searchSortOrder, facilityIds, query, false, fields);
+      return NurseryBatchService.getAllBatches(
+        organizationId,
+        searchSortOrder,
+        facilityIds,
+        undefined,
+        query,
+        false,
+        fields
+      );
```

Comment:
would it be better to make these non-positional, maybe an object instead? Guess it might touch a few places..

Reply:
Yea I don't love the signature on this one, but felt out of scope to refactor for this. Ideally all of these pre-baked Search API calls would have the same signature...